# Python Programming Examples


1. Python program to add two numbers
2. Maximum of two numbers in Python
3. Python Program for factorial of a number
4. Python Program for simple interest
5. Python Program for compound interest
6. Python Program to check Armstrong Number
7. Python Program for Program to find area of a circle
8. Python program to print all Prime numbers in an Interval
9. Python program to check whether a number is Prime or not
10. Python Program for n-th Fibonacci number
11. Python Program for How to check if a given number is Fibonacci number?
12. Python Program for n\’th multiple of a number in Fibonacci Series
13. Program to print ASCII Value of a character
14. Python Program for Sum of squares of first n natural numbers
Python Program for cube sum of first n natural numbers


#  Python program to add two numbers

In [2]:
a =4
b =7
print("adding without third variable")
print(a+b)
print("using the third variable")
c= a+b
print(c)

adding without third variable
11
using the third variable
11



wap to taking the input a, b from the user  and assign the value in variable c



In [5]:
a= int(input("enter the input for a: "))
b= int(input("enter the input for b: "))

c=a+b
print("after adding the value a and b =",c)

after adding the value a and b = 64


# WAP Maximum of two numbers in Python

In [6]:
a=2
b =3
if a>b:
    print("a is greater")
else:
    print("b is greater")

b is greater


# take two input and compare the two number abd check which number is greater

In [7]:
a = int(input("enter value for a: "))
b = int(input("enter value for b: "))

if a>b:
    print("a is greater")
else:
    print("b is greater")

b is greater


# Python Program for factorial of a number


In [11]:
# pyhton program to find factorial
def fact(n):
    return 1 if (n==1 or n==0) else n * fact(n-1)


num =6
print("factorial of ", num ,"is",fact(num))

factorial of  6 is 720


In [13]:
# user use factorial of number

def fact(n):
    if n<0:
        return 0
    elif n == 0 or n ==1:
        return 1
    else:
        fact = 1
        while (n > 1):
            fact = fact*n
            n=n-1
        return fact
    
#drive code
num =5

print("the factorial of ", num, "=",fact(num
                                         ))
        

the factorial of  5 = 120


In [15]:
def fact(n):
    if n<0:
        return 0
    elif n == 0 or n ==1:
        return 1
    else:
        fact = 1
        while (n > 1):
            fact = fact*n
            n=n-1
        return fact
    
#drive code
num =int(input("enter the value for n: "))

print("the factorial of ", num, "=",fact(num))


the factorial of  8 = 40320


In [17]:
# By using the inbuilt function.

import math 
def fact(n):
    return (math.factorial(n))

num =7
print(fact(num))

5040



## write a  Python Program for simple interest
~~~

Simple interest formula is given by:
Simple Interest = (P x T x R)/100
Where,
P is the principle amount
T is the time and
R is the rate
~~~

In [21]:
p = int(input("enter the value for principal:"))
r= int(input("enter the time period:"))
t = int(input("enter the rate of interest: "))

si = (p*r*t)/100
print("Principal: ", p)
print("Rate :",r)
print("time: ",t)
print("Simple Intesrest= ",si)

Principal:  10000
Rate : 8
time:  4
Simple Intesrest=  3200.0


In [25]:
def si(p,r,t):
    print('The principal :',p)
    print('The rate :',r)
    print('The time :',t)
    sinterest = (p*r*t)/100
    print("the simple iterest:", sinterest)
    return sinterest

p = 9000
r= 8
t =4

a=si
(p,r,t)

The principal : 9000
The rate : 8
The time : 4
the simple iterest: 2880.0


# write a Python Program for compound interest
~~~

Formula to calculate compound interest annually is given by: 
A = P(1 + R/100) t 
Compound Interest = A – P 
Where, 
A is amount 
P is principle amount 
R is the rate and 
T is the time span


~~~

In [37]:
def c(p,r,t):
    amt = p * (pow((1+r/100),t))
    ci = amt - p
    
    
    print("Amount:",amt)
    print("principal: ",p)
    print("time: ",t)
    print("rate: ",r)
    print(" Compound interest: ",ci)
    
c(1000,10.23,5)
                     

Amount: 1627.4177072746775
principal:  1000
time:  5
rate:  10.23
 Compound interest:  627.4177072746775


# write a Python Program to check Armstrong Number
~~~

Given a number x, determine whether the given number is Armstrong number or not. A positive integer of n digits is called an Armstrong number of order n (order is number of digits) if.

abcd... = pow(a,n) + pow(b,n) + pow(c,n) + pow(d,n) + .... 
Example:

Input : 153
Output : Yes
153 is an Armstrong number.
1*1*1 + 5*5*5 + 3*3*3 = 153

Input : 120
Output : No
120 is not a Armstrong number.
~~~

In [39]:
# take input from the user 
num = int(input("enter a nuumber: "))
# initialize sum
sum = 0 
# find the sum of of the cube of each digit
temp = num

while temp >0:
    digit = temp %10
    sum = sum + digit ** 3
    temp //=10
    
# display the result 

if num == sum:
    print(num, "is a Armstrong number")
else:
    print(num,"is not a Armstrong Number") 

407 is a Armstrong number


# Python Program for Program to find area of a circle

Area of a circle can simply be evaluated using following formula.

Area = pi * r2
where r is radius of circle 

In [41]:
def area(r):
    pi =3.14
    a=pi*(r*r)
    print("Area of rectangle: ",a)
    
    
    
radius = int(input("enter radius "))
area(radius)


Area of rectangle:  6358.5
